dyson/chain.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "dyson",
+  "website": "https://dysonprotocol.com/",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Dyson Protocol",
+  "chain_id": "dyson-mainnet-01",
+  "bech32_prefix": "dys",
+  "node_home": "$HOME/.dyson",
+  "daemon_name": "dysond",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "dys",
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.01
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "dys"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://gitlab.com/dysonproject/dyson",
+    "recommended_version": "v0.0.0-rc1",
+    "compatible_versions": [
+      "v0.0.0-rc1"
+    ],
+    "genesis": {
+      "genesis_url": "https://gitlab.com/dysonproject/dyson-deploy/-/raw/develop/genesis.json"
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      },
+      {
+        "id": "1f3bd1f48c7c277e7be83895cdbb46f8333adb61",
+        "address": "178.79.183.171:26656",
+        "provider": ""
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      },
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://dys-tm.dyson.lol:443",
+        "provider": "lol"
+      },
+      {
+        "address": "https://dys-tm.dysonprotocol.com:443",
+        "provider": "dysonprotocol"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://dys-api.dyson.lol:443",
+        "provider": "lol"
+      },
+      {
+        "address": "https://dys-api.dysonprotocol.com:443",
+        "provider": "dysonprotocol"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "https://dys-grpc.dyson.lol:443",
+        "provider": "lol"
+      }
+    ],
+    "evm-http-jsonrpc": []
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.dys.dysonprotocol.com/dyson/",
+      "tx_page": "https://explorer.dys.dysonprotocol.com/dyson/tx/${txHash}"
+    },
+    {
+      "kind": "ping.pub",
```

Comment:
also, should probably change these "kind"/providers? as 'lol' and 'dysonprotocol' maybe?

Reply:
thanks, I wasn't certain what to put there